src/psim/simulations/single_orbit.cpp

```diff
@@ -22,7 +22,7 @@
 // SOFTWARE.
```

Comment:
What is the reason for keeping this model around, now that the attitude-orbit model exists?

Reply:
If I end up running orbit estimator testing and don't want/need all the attitude simulation stuff involved. This is actually exactly why I wanted to shift to this type of architecture lol